docs/api/security/security.md

```diff
@@ -2,10 +2,20 @@
 
 Security on Arm Mbed OS is divided into the following parts:
 
-- Platform Security Architecture (PSA). For information about working with PSA in the context of Mbed OS, please see [Mbed PSA asset protection](../apis/psa-api.html).
+- [PSA SPM](./spm.md) - used for accessing secure services within Secure Processing environment (on PSA targets only)
 
-    For full details, please see the [PSA site](https://developer.arm.com/products/architecture/security-architectures/platform-security-architecture).
+- [PSA internal storage](./psa_internal_storage.md) - used to save PSA RoT state
```

Comment:
"PSA internal storage - used to save PSA RoT state" - Is this correct? From reading the documentation, I would have thought the RoT state is handled by PSA Lifecycle.